docs/examples/good_practices/wandb_setup/index.rst

```diff
@@ -0,0 +1,45 @@
+WANDB Setup
+=====================================
+
+
+Prerequisites:
+
+* :doc:`/examples/frameworks/pytorch_setup/index`
+* :doc:`/examples/distributed/single_gpu/index`
+
+Make sure to create a WANDB account, then you can either :
+- Set your WANDB_API_KEY environment variable
+- Run `wandb login` from the command line
+
+* `<https://wandb.ai/site>`_
+
+Other interesting resources:
+
+* `<https://docs.wandb.ai/quickstart>`_
+
+Click here to see `the source code for this example
+<https://github.com/mila-iqia/mila-docs/tree/master/docs/examples/good_practices/wandb_setup>`_
```

Comment:
Hello ! Je crois que ce lien mène vers une page Github 404 (not found).

Si nécessaire, on peut voir le rendu de cet exemple ici (le lien est disponible dans les résultats CI de chaque commit de la PR): https://mila-docs--199.org.readthedocs.build/en/199/examples/good_practices/wandb_setup/index.html#

Reply:
Oh, ok ! Donc c'est vraiment un lien direct vers Github, pas un lien vers une section de la doc.

Dans ce cas, c'est bon pour moi ! @lebrice @gyom